CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+# 1.4.1 (October 26th, 2023)
+
 CHANGES:
 
 * Build with Go v1.21.3
```

Comment:
Please can you add an entry for the gRPC bump I just merged in #227 as well? Sorry for the late addition, but it seemed worth including in the release as it includes a fix for the same CVE being fixed by our Go bump, so could still result in scanner noise otherwise.

Reply:
Sure! Will do.